addons/rose/app/styles/rose/components/layout/_index.scss

```diff
@@ -15,5 +15,5 @@
 // but used here reflects the fact that the "main" element, of which there
 // can only be one per page, generally has padding in Structure layouts.
 main {
-  padding: sizing.rems(l);
+  padding: 1.5rem 3rem;
```

Comment:
Stop using `sizing.rems` because I understand this is something we want to deprecate. But please, let me know if we should still using `sizing.rems`

Reply:
I think you are right.

Reply:
to give some background- `sizing.rems` is part of the rose addon and that's why we stopped using this when HDS components were used. But isn't layout currently a rose component?

Reply:
I'm okay with removing it, can we also remove L6 since we won't need it?

Reply:
@DhariniJeeva Awesome catch!! thank you!!